Neos.Neos/Classes/Routing/NodeIdentityConverterAspect.php

```diff
@@ -45,15 +46,13 @@ class NodeIdentityConverterAspect
      */
     public function convertNodeToContextPathForRouting(JoinPointInterface $joinPoint): array|string
     {
+        // todo whyyyy does `Route::resolves` even call `convertObjectsToIdentityArrays`? And why nodes?
```

Comment:
```suggestion
        // todo whyyyy does `Route::resolves` even call `convertObjectsToIdentityArrays`? And why nodes?
```
What's the TODO here?

Reply:
yeah ill remove them again, maybe you could explain that thing to me and if we can remove it :D 
-> discussion: https://github.com/neos/neos-development-collection/issues/5069